tests/test_netcdf_monitor.py

```diff
@@ -25,6 +26,67 @@
 }
 
 
+class NetCDFMonitorAliasTests(unittest.TestCase):
+
+    def setUp(self):
+        self.ncfile = 'out.nc'
+
+    def tearDown(self):
+        if os.path.isfile(self.ncfile):
+            os.remove(self.ncfile)
+
+    def store_state_and_check_file(self, aliases):
+        assert not os.path.isfile(self.ncfile)
+        monitor = NetCDFMonitor(self.ncfile, aliases=aliases,
+                                write_on_store=True)
+        monitor.store(state)
+        assert os.path.isfile(self.ncfile)
+
+    def check_nc_var(self, varname, varunit):
```

Comment:
Should also check that the 'long_name' attribute is set to the right value when an alias is used (I suspect the current code might fail that test). It's not really defined what we should do when an alias is not used, so you don't need to test long_name in that case.